util/encoding/encoding.go

```diff
@@ -144,14 +144,14 @@ func write64(wc io.WriteCloser, data ...kyber.Marshaling) error {
 	return wc.Close()
 }
 
-func getHex(r io.Reader, len int) ([]byte, error) {
```

Comment:
I'll go and do penitence - that was mine...

Reply:
```
r := rules.New("Don't shadow keywords.")
r.Penance = func () { for range dedis.Engineers() { 🍰.buy() } }
```